nixos/modules/services/hardware/udev.nix

```diff
@@ -66,6 +66,7 @@ let
           --replace \"/sbin/blkid \"${pkgs.util-linux}/sbin/blkid \
           --replace \"/bin/mount \"${pkgs.util-linux}/bin/mount \
           --replace /usr/bin/readlink ${pkgs.coreutils}/bin/readlink \
+          --replace /usr/bin/cat ${pkgs.coreutils}/bin/cat \
```

Comment:
AFAIK `--replace` is deprecated ( https://github.com/NixOS/nixpkgs/issues/356002 ), could we switch to `--replace-quiet`? That should be the same behavior, but non deprecated. If I understand the code snippet correct we are traversing all kind of possible udev rules, so no one knows if all replacements are existing.

Reply:
that is a valid point, but should it be addressed in this PR?

Reply:
For me it's valid to do some chore work aside touching code, tho I would place it in a separate commit.
But I am also just a random person :)

Reply:
fair enough, actually :)